lib/node_modules/@stdlib/stats/base/svariancewd/docs/repl.txt

```diff
@@ -1,9 +1,9 @@
 
-{{alias}}( N, correction, x, stride )
+{{alias}}( N, correction, x, strideX )
     Computes the variance of a single-precision floating-point strided array
     using Welford's algorithm.
 
-    The `N` and `stride` parameters determine which elements in `x` are accessed
+    The `N` and stride parameters determine which elements in x are accessed
```

Comment:
```suggestion
    The `N` and stride parameters determine which elements in the strided array are accessed
```

You will likely need to rewrap the lines. The point of removing references to `x` to avoid copy-paste errors in future packages.